src/performance.ts

```diff
@@ -9,11 +10,11 @@ export const performance = {
     clearMarks: () => {},
     clearMeasures: () => {},
     clearResourceTimings: () => {},
-    getEntries: () => [],
-    getEntriesByName: () => [],
-    getEntriesByType: () => [],
-    getMarks: () => [],
-    getMeasures: () => [],
+    getEntries: () => [] as Array<typeof performanceResourceTimingStub>,
+    getEntriesByName: () => [] as Array<typeof performanceResourceTimingStub>,
+    getEntriesByType: () => [] as Array<typeof performanceResourceTimingStub>,
+    getMarks: () => [] as Array<typeof performanceEntryStub>,
+    getMeasures: () => [] as Array<typeof performanceEntryStub>,
```

Comment:
Why can't these be like `getMeasures: (): Array<typeof performanceEntryStub> => []`?

Reply:
There's a general preference for the return type annotation (more type safe than coercion in general), so if we can go for that let's do it.